src/Components/AllPosts.js

```diff
@@ -0,0 +1,68 @@
+import { StyledPost } from './styles/Post.styled'
+
+import { useSelector, useDispatch } from 'react-redux'
+
+import { useEffect } from 'react'
+
+import { getAllPosts } from '../Features/postsSlice'
+
+import moment from 'moment'
```

Comment:
`Moment.js is a legacy project, now in maintenance mode. In most cases, you should choose a different library.`
- [Moment JS NPM](https://www.npmjs.com/package/moment)

Reply:
Thankyou, didnt know this. Changed to date fns, works fine.